src/modules/authentication/AuthenticationWrapper.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  AccountInfo,
+  AuthenticationResult, InteractionRequiredAuthError,
+  PopupRequest, SilentRequest
+} from '@azure/msal-browser';
+
+import { AUTH_URL, DEFAULT_USER_SCOPES, HOME_ACCOUNT_KEY } from '../../app/services/graph-constants';
+import { geLocale } from '../../appLocale';
+import { getCurrentUri } from './authUtils';
+import IAuthenticationWrapper from './IAuthenticationWrapper';
+import { msalApplication } from './msal-app';
+
+const defaultScopes = DEFAULT_USER_SCOPES.split(' ');
+const homeAccountKey = HOME_ACCOUNT_KEY;
+
+export class AuthenticationWrapper implements IAuthenticationWrapper {
+
+  private static instance: AuthenticationWrapper;
+  private isConsentFlow: boolean = false;
```

Comment:
Please add a summary explaining the role of the properties and functions.

Reply:
@nikithauc I tried to limit using comments by instead using variable and function names that could be self-documenting; Unless when documenting a major decision eg the `clearCache()` function. Does this mean the properties and functions do not convey the meaning?

Reply:
Yes, I didn't understand what `isConsentFlow` means?